src/more-infos/more-info-media_player.html

```diff
@@ -45,6 +45,15 @@
         margin-left: 10px;
       }
 
+      iron-icon.sound-mode-input {
```

Comment:
Same as source-input. You should combine them

Reply:
I combined both